tests/conftest.py

```diff
@@ -13,9 +13,9 @@
     Platform,
     create_platform,
 )
-from qibolab.platform.load import PLATFORMS
-from qibolab.sequence import PulseSequence
-from qibolab.sweeper import ParallelSweepers, Parameter, Sweeper
+from qibolab._core.platform.load import PLATFORMS
+from qibolab._core.sequence import PulseSequence
+from qibolab._core.sweeper import ParallelSweepers, Parameter, Sweeper
```

Comment:
This is explained in https://github.com/qiboteam/qibolab/issues/949 rule 2. (in particular, it's pretty explicit in the example)

Reply:
Strictly speaking tests are outside the qibolab package, so I am not sure if rule 2 applies (if I understand it correctly). However, I see that `qibolab._core` is used in various occasions in tests and in some cases we even have to use it, since we also need to test private objects. Therefore, it should be fine to keep.